### DISCUSSION TITLE
DeploymentNode belonging to two other

### DISCUSSION THREAD

**Discussion by lasagnesmatinmidisoir** [structurizr/java#270] [Q&A] [answered]
Hi,

In the case of a service with high availability, thus multiple instances deployed on different subnets, I would like to have two DeploymentNodes for the subnets, one for the service and two Infrastructure Nodes for its two instances.
The DeploymentNode for the service would belong to the two subnets.
How can I achieve this in Structurizr for Java ?

![image](https://github.com/structurizr/java/assets/162685569/f45fee4e-2a62-4cd1-87f9-73b1e51b62f3)

Best,

**Comment by simonbrowndotje** [comment, marked as answer]
This isn't supported I'm afraid.
